README.md

```diff
@@ -114,51 +114,39 @@ meant to be tracked. (Note: PolyTracker can in fact track multiple input files
 
-The instrumented software will write its output to the path specified in `POLYDB`, or `polytracker.db` if omitted.
-This is a sqlite3 database that can be operated on by running:
+The instrumented software will write its output to the path specified in `POLYDB`, or `polytracker.tdag` if omitted.
+This is a binary file that can be operated on by running:
 ```python
 from polytracker import PolyTrackerTrace
 
-trace = PolyTrackerTrace.load("polytracker.db")
-
-for event in trace:
-    # this prints every single event in the trace, in order
-    print(event)
-
-for function in trace.functions:
-    # this returns a function object for every function observed
-    # (this is different than a function invocation, which we describe below)
-    print(function.demangled_name)
-
-main_func = trace.get_function("main")
-for taint in main_func.taints().regions():
-    # this iterates over every taint touched by the main function aggregated across all invocations of main
-    print(f"source={taint.source}, offset={taint.offset}, length={taint.length}, value={taint.value}")
-```
-Individual function invocations in the trace can also be enumerated:
-```python
-entrypoint = trace.entrypoint
-# entrypoint is a function invocation object associated with a single invocation of the entrypoint of the trace (main)
-print(f"Entrypoint is: {entrypoint!s}")
-for called_function in entrypoint.calls():
-    # called_function is another function invocation object
-    # associated with a function called from entrypoint
-    print(str(called_function))
-    for region in called_function.taints().regions():
-        # this will iterate over every tainted region operated on by called_function,
-        # including any functions called to from called_function
-        print(f"\t{region.value}")
+trace = PolyTrackerTrace.load("polytracker.tdag")
+# TODO(facutuesca): replace with example using new API
+
+# for event in trace:
+#     # this prints every single event in the trace, in order
+#     print(event)
+# 
+# for function in trace.functions:
+#     # this returns a function object for every function observed
+#     # (this is different than a function invocation, which we describe below)
+#     print(function.demangled_name)
+# 
+# main_func = trace.get_function("main")
+# for taint in main_func.taints().regions():
+#     # this iterates over every taint touched by the main function aggregated across all invocations of main
+#     print(f"source={taint.source}, offset={taint.offset}, length={taint.length}, value={taint.value}")
 ```
 
 You can also run an instrumented binary directly from the REPL:
 ```python
 $ polytracker
-PolyTracker (3.0.0)
+PolyTracker (3.1.0)
 https://github.com/trailofbits/polytracker
 Type "help" or "commands"
 >>> trace = run_trace("path_to_binary", "path_to_input_file")
->>> for event in trace:
-...     print(event)
+# TODO(facutuesca): Replace with working example new API
+# >>> for event in trace:
+# ...     print(event)
 ```
```

Comment:
Consider only including one example since we dropped the basic block/function tracing and currently only have one use case.

Reply:
I removed the commented out section, but left the `run_trace` line to have at least one example of the REPL being used